PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="MD003">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Unidentified prayer in the name of all saints and martyrs</title>
```

Comment:
Is that a useful name for a work? I mean, all the prayers were previously unidentified before they were identified. By creating this record with long incipit and explicit, you have done a lot of work to identify them.